test/functional/tests/ci/test_ci_write_mode_write_around.py

```diff
@@ -0,0 +1,212 @@
+#
+# Copyright(c) 2022 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+#
+
+import pytest
+import time
+
+from api.cas import casadm
+from core.test_run import TestRun
+from storage_devices.disk import DiskType, DiskTypeSet, DiskTypeLowerThan
+from test_utils.size import Unit, Size
+from test_tools.dd import Dd
+from test_tools.iostat import IOstatBasic
+
+
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.nand, DiskType.optane]))
+@pytest.mark.require_disk("core", DiskTypeLowerThan("cache"))
+@pytest.mark.CI()
+def test_ci_write_mode():
+    """
+    title: Verification test for write mode: write around
+    description: Verify if write around mode works as expected and caches only reads
+    and does not cache write
+    pass criteria:
+    - writes are not cached
+    """
+
+    with TestRun.step("Prepare devices"):
+        cache_device = TestRun.disks["cache"]
+        core_device = TestRun.disks["core"]
+
+        cache_device.create_partitions([Size(1024, Unit.MebiByte)])
+        core_device.create_partitions([Size(2048, Unit.MebiByte)])
+
+        cache_device = cache_device.partitions[0]
+        core_device = core_device.partitions[0]
+
+    with TestRun.step("Test read from cache"):
```

Comment:
This is one huge test step. Please try to split it into few smaller steps so that it will be clear in the log what is going on in the test.